lib/sprockets/commoner.rb

```diff
@@ -10,8 +11,10 @@ def self.sprockets4?
     end
   end
 
+  register_mime_type "application/jsx", extensions: [".js.jsx",".jsx"], charset: :unicode
   register_postprocessor 'application/javascript', ::Sprockets::Commoner::Processor
   register_transformer 'application/json', 'application/javascript', ::Sprockets::Commoner::JSONProcessor
+  register_transformer 'application/jsx', 'application/javascript', ::Sprockets::Commoner::JSXProcessor
```

Comment:
Not much for.. it just tells the sprockets that 'application/jsx' can be transformed into 'application/javascript' so it can be later included in the 'js' files.

Or we could just tell it that '.jsx' files are '.js' files stright away , by:

```
register_mime_type "application/javascript", extensions: [".js.jsx",".jsx"], charset: :unicode
```

Or maybe there's some another better way to do it. I'm not sure, I'm not really familiar with sprockets yet